tests/common.py

```diff
@@ -0,0 +1,124 @@
+import logging
+import json
+import os
+
+import pyarrow as pa
+from pyarrow.lib import tobytes
+from pyarrow.lib import Table
```

Comment:
```suggestion
```
I would make use of `pa` here, otherwise `Table` seems pretty generic.  And there's a `pa.Table` available that points to `pa.lib.Table`

Reply:
Agree, in the public API we tend to use `from pyarrow import Table`